test/image/mocks/marker_symbols.json

```diff
@@ -0,0 +1,99 @@
+{
+  "data": [
+    {
+      "x": [1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9],
+      "y": [1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 4, 4, 4, 4, 4, 4, 4, 4, 4, 5, 5, 5, 5, 5, 5, 5, 5, 5],
+      "mode": "markers",
+      "text": [
+        "marker number: 0<br>marker string: 'circle'",
```

Comment:
Good catch! Can you include just a few examples of 1xx (`-open`), 2xx (`-dot`) and 3xx (`-open-dot`) in this mock? It should be able to support just *one* more row 😉 

And then we'll be able to see how this compares with the gl implementation, where I'm pretty sure colors are handled differently in the `-open` case than they currently are for svg, ie we would never have a double line like gl2d currently has.